specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json

```diff
@@ -0,0 +1,738 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ExpressRouteCrossConnection."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting ExpressRouteCrossConnection resource.",
+            "schema": {
+              "$ref": "#/definitions/ExpressRouteCrossConnection"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ExpressRouteCrossConnections"
+        ],
+        "operationId": "ExpressRouteCrossConnections_Put",
+        "description": "Update the specified ExpressRouteCrossConnection.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "crossConnectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ExpressRouteCrossConnection."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+            "$ref": "#/definitions/ExpressRouteCrossConnection"
+            },
+            "description": "Parameters supplied to the update express route crossConnection operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting ExpressRouteCrossConnection resource."
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+        "ExpressRouteCrossConnections"
+        ],
+        "operationId": "ExpressRouteCrossConnections_UpdateTags",
+        "description": "Updates an express route cross connection tags.",
+        "parameters": [
+        {
+          "name": "resourceGroupName",
+          "in": "path",
+          "required": true,
+          "type": "string",
+          "description": "The name of the resource group."
+        },
+        {
+          "name": "crossConnectionName",
+          "in": "path",
+          "required": true,
+          "type": "string",
+          "description": "The name of the cross connection."
+        },
+        {
+          "name": "parameters",
+          "in": "body",
+          "required": true,
+          "schema": {
+          "$ref": "./network.json#/definitions/TagsObject"
```

Comment:
nit: fix indentation
Shouldn't this parameter be named `tags` instead?

Reply:
Fixed indentation. The naming is consistent with corresponding operation on ExpressRoute Circuit

Reply:
Thank you for helping with the reviews.

Reply:
`parameters` is still to generic, please name it something more meaningful